ims/ostis_tech/semantic_network_represent/section_sc_code/section_subjdomain_numbers_and_numeric_structures/content_html/def_subjdomain_numbers_and_numeric_structures_exercise_1.html

```diff
@@ -0,0 +1 @@
+<p>Представьте следующее утверждение на языке Scg: "Отрезок DM равен одной третьей отрезка DE."</p>
```

Comment:
Most of file name already exist in a file path (directories name). I think it should be shorter